### DISCUSSION TITLE
Is there a way to visually select the new comment text object?

### DISCUSSION THREAD

**Discussion by carschandler** [neovim/neovim#28901] [Q&A]
Really love the addition of comment support to stock nvim! Was also very excited to see the `gc` text object added to operator-pending mode. It would be great if there were an easy way to visually select this object... I'm not sure if there's a way to visually select an operator-pending object by default or if this would require some new mapping like `vigc` (having a hard time coming up with this one since `vgc` would be the natural choice, but we have an interesting case where the text object mapping is the same as a visual-mode action, so this doesn't work).

**Comment by frolvanya** [comment]
If I am not mistaken, right now you can do it only with plugin. For example, https://github.com/nvim-treesitter/nvim-treesitter-textobjects and custom mapping for `@comment.inner` and `@comment.outer`
